<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateNew</class>
 <widget class="QDialog" name="CreateNew">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>生成新练习</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="group_page">
       <property name="title">
        <string>页面设置</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>纸型：</string>
          </property>
          <property name="buddy">
           <cstring>cmbPageSize</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbPageSize">
          <item>
           <property name="text">
            <string>A4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>B5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>A3</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>左边距：</string>
          </property>
          <property name="buddy">
           <cstring>spbLeftPadding</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spbLeftPadding">
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>毫米</string>
          </property>
          <property name="buddy">
           <cstring>spbLeftPadding</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>顶边距：</string>
          </property>
          <property name="buddy">
           <cstring>spbTopPadding</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spbTopPadding">
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>毫米</string>
          </property>
          <property name="buddy">
           <cstring>spbTopPadding</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>右边距：</string>
          </property>
          <property name="buddy">
           <cstring>spbRightPadding</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spbRightPadding">
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>毫米</string>
          </property>
          <property name="buddy">
           <cstring>spbRightPadding</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>底边距：</string>
          </property>
          <property name="buddy">
           <cstring>spbBottomPadding</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spbBottomPadding">
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>毫米</string>
          </property>
          <property name="buddy">
           <cstring>spbBottomPadding</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="group_label">
       <property name="title">
        <string>内容设置</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>起始单元：</string>
              </property>
              <property name="buddy">
               <cstring>startUnit</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="startUnit">
              <item>
               <property name="text">
                <string>1.1</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>结束单元：</string>
              </property>
              <property name="buddy">
               <cstring>endUnit</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="endUnit">
              <item>
               <property name="text">
                <string>1.1</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>列数：</string>
              </property>
              <property name="buddy">
               <cstring>spbColumns</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spbColumns">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>5</number>
              </property>
              <property name="value">
               <number>2</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>列间距：</string>
              </property>
              <property name="buddy">
               <cstring>spbColMargin</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spbColMargin">
              <property name="value">
               <number>5</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>毫米</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="printPageNumber">
              <property name="text">
               <string>打印页码</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="isRandom">
              <property name="text">
               <string>乱序打印</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>打印内容：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="selectWords">
              <property name="text">
               <string>单词</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="selectWordGroup">
              <property name="text">
               <string>词组</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="printCh">
              <property name="text">
               <string>打印中文</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="printEn">
              <property name="text">
               <string>打印英文</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="createAnswer">
              <property name="text">
               <string>生成答案</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_19">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>行高：</string>
              </property>
              <property name="buddy">
               <cstring>lineHeight</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="lineHeight">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>毫米</string>
              </property>
              <property name="buddy">
               <cstring>lineHeight</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="d_page">
              <property name="text">
               <string>双面</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="onlyError">
              <property name="text">
               <string>只选错词</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCreate">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>生成PDF</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>58</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOpenPDF">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>打开PDF</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>58</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>CreateNew</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>806</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
